update.md

```diff
@@ -0,0 +1,54 @@
+# ESI demo environment setup
+## Requirements
+* CentOS 8
+* Ussuri tripleo-repos
+## Steps
+* Download and install the python-tripleo-repos RPM
+```
+sudo dnf install -y https://trunk.rdoproject.org/centos8/component/tripleo/current/python3-tripleo-repos-0.1.1-0.20200612051255.ecf6206.el8.noarch.rpm
+```
+* Enable the current Ussuri repositories
+```
+sudo -E tripleo-repos -b ussuri current
+```
+* Install the TripleO CLI
+```
+sudo yum install -y python3-tripleoclient
+```
+* git clone esi-sandbox repo and cd esi-sandbox
+* Open deploy.sh, change INTERFACE=eth0 to INTERFACE=ens10; comment out ceph-ansible related commands.
```

Comment:
This is going to be environment dependent; maybe just a note that a user may have to change INTERFACE in deploy.sh?

Reply:
Make sense!